scipy/linalg/tests/test_basic.py

```diff
@@ -762,15 +762,19 @@ def test_singularity(self):
         b = np.arange(9)[:, None]
         assert_raises(LinAlgError, solve, a, b)
 
-    def test_ill_condition_warning(self):
-        a = np.array([[1, 1, 1],
-                      [1+1e-16, 1-1e-16, 1],
-                      [1-1e-16, 1+1e-16, 1],
-                      ])
-        b = np.ones(3)
-        with warnings.catch_warnings():
```

Comment:
The lint error is just about `warning` no longer being used; I'll remove the import.